packages/apps/ability-aave/src/lib/helpers/aave.ts

```diff
@@ -19,19 +19,62 @@ import {
   AaveV3OptimismSepolia,
   AaveV3ScrollSepolia,
 } from '@bgd-labs/aave-address-book';
-import { encodeFunctionData } from 'viem';
+import {
+  VINCENT_CONTRACT_ADDRESS_BOOK,
+  FEE_DIAMOND_ABI,
+} from '@lit-protocol/vincent-contracts-sdk';
+import { encodeFunctionData, getAddress } from 'viem';
 
 import { ERC20_ABI } from './erc20';
 
 const ZERO_VALUE: Hex = '0x0';
 
+/**
+ * Fee Contract ABI for Aave operations
+ * These functions route through the fee contract instead of directly to Aave
+ */
+export const FEE_CONTRACT_ABI: Abi = FEE_DIAMOND_ABI as Abi;
+
 export interface Transaction {
   data: Hex;
   from: Address;
   to: Address;
   value: Hex;
 }
 
+/**
+ * Chain ID to fee contract chain name mapping
+ */
+const CHAIN_ID_TO_FEE_CHAIN_NAME: Record<number, keyof typeof VINCENT_CONTRACT_ADDRESS_BOOK.fee> = {
```

Comment:
we should prob fix the fee contract stuff so this isn't needed.  let me see if i can fix and publish a new version